src/quo/components/inputs/address_input/style.cljs

```diff
@@ -35,7 +36,8 @@
          :color        (colors/theme-colors colors/neutral-100 colors/white theme)
          :margin-top   (if platform/ios? 0 -4)
          :margin-right 8
-         :height       (if platform/ios? 24 40)))
+         :max-height   (+ (* input-height 2) (when platform/ios? 1))
```

Comment:
Why do we have this odd number ? Why add 1 for ios? 

Can this just be `:max-height 48` ?

Reply:
@shivekkhurana actually not needed, updated it 👍